src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -19,6 +19,9 @@
  */
 public class CommitResultAggregator {
 
+    private static final int DAYS_IN_MS = 24 * 60 * 60 * 1000;
+    private static Date lastDate = null;
```

Comment:
What is the assumption that the lastDate can be a global variable here?
Perhaps make it a returning value in the methods through pointer etc.